lib/index.ts

```diff
@@ -137,7 +151,7 @@ export default class Backburner {
     return current;
   }
 
-  public end() {
+  public end(fromAutorun = false) {
```

Comment:
I would prefer to not expose this flag via a public API, we should maybe create `_end` that can have w/e flag, but if at all possible keep the existing one pristine (if possible).

Reply:
Yep, totally makes sense.